packages/components/src/Typography/css/Truncate.css

```diff
@@ -0,0 +1,6 @@
+.textTruncate {
+  display: -webkit-box !important;
```

Comment:
does this `!important` need to be here? If so can we maybe throw in a comment as to why?

Reply:
[This](https://developer.mozilla.org/en-US/docs/Web/CSS/-webkit-line-clamp) doc says that for webkit-line-clamp to work the display should be set to `webkit-box`. So thought of enforcing that display is always set to this value.

Reply:
If that makes sense, I can add that as a comment there.

Reply:
sounds good!